.github/workflows/continuous-integration.yml

```diff
@@ -37,15 +36,15 @@ jobs:
 
       - name: "Install PHP with XDebug"
         uses: "shivammathur/setup-php@v2"
-        if: "${{ matrix.php-version == '7.1' }}"
+        if: "${{ matrix.php-version == '7.2' }}"
         with:
           php-version: "${{ matrix.php-version }}"
           coverage: "xdebug"
           ini-values: "zend.assertions=1"
```

Comment:
You can drop this block entirely. Alternatively, you can migrate to reusing a centralized workflow, like this: https://github.com/doctrine/common/pull/937/files